django/template/base.py

```diff
@@ -122,6 +122,9 @@ def __init__(self, name, template_name=None, loader=None):
     def __str__(self):
         return self.name
 
+    def __repr__(self):
+        return f'{self.__class__.__qualname__}(name={self.name!r})'
```

Comment:
To be more consistent with other `__repr__()` methods I would use:
```suggestion
        return '<%s name=%r>' % (self.__class__.__qualname__, self.name)
```

Reply:
@felixxm Thanks for the suggestion, I will make the change!